rising/transforms/affine.py

```diff
@@ -37,55 +36,15 @@ def __init__(self, scale: AffineParamType = None,
 
         Parameters
         ----------
-        scale : torch.Tensor, int, float, optional
-            the scale factor(s). Supported are:
-                * a full transformation matrix of shape
-                    (BATCHSIZE x NDIM x NDIM)
-                * a single parameter (as float or int), which will be
-                    replicated for all dimensions and batch samples
-                * a single parameter per sample (as a 1d tensor), which will
-                    be replicated for all dimensions
-                * a single parameter per dimension (either as 1d tensor or as
-                    2d transformation matrix), which will be replicated for
-                    all batch samples
-            None will be treated as a scaling factor of 1
-        rotation : torch.Tensor, int, float, optional
-            the rotation factor(s). Supported are:
-                * a full transformation matrix of shape
-                    (BATCHSIZE x NDIM x NDIM)
-                * a single parameter (as float or int), which will be
-                    replicated for all dimensions and batch samples
-                * a single parameter per sample (as a 1d tensor), which will
-                    be replicated for all dimensions
-                * a single parameter per dimension (either as 1d tensor or as
-                    2d transformation matrix), which will be replicated for
-                    all batch samples
-            None will be treated as a rotation factor of 1
-        translation : torch.Tensor, int, float
-            the translation offset(s). Supported are:
-                * a full homogeneous transformation matrix of shape
-                    (BATCHSIZE x NDIM+1 x NDIM+1)
-                * a single parameter (as float or int), which will be
-                    replicated for all dimensions and batch samples
-                * a single parameter per sample (as a 1d tensor), which will
-                    be replicated for all dimensions
-                * a single parameter per dimension (either as 1d tensor or as
-                    2d transformation matrix), which will be replicated for
-                    all batch samples
-            None will be treated as a translation offset of 0
         matrix : torch.Tensor, optional
             if given, overwrites the parameters for :param:`scale`,
```

Comment:
There are no parameters for scale, rotation and translation here. Either they are missing or (more likely) the doctoring is not up to date

Reply:
See below